bayesian/api_v1.py

```diff
@@ -403,82 +398,7 @@ def get(external_request_id):
             }
 
         for stack in stacks:
-            user_stack_deps = stack.get('user_stack_info', {}).get('dependencies', [])
-            for dep in user_stack_deps:
-                if user_stack_sentiment_result is not None:
-                    user_stack_sentiment_item = \
-                        get_item_from_list_by_key_value(user_stack_sentiment_result.get(
-                            'task_result', {}).get('sentiment', []), 'manifest_file_path',
-                            stack.get('manifest_file_path'))
-                    dep['sentiment']['overall_score'] = \
-                        user_stack_sentiment_item.get(dep['name'], {}).get('score', 0)
-                    dep['sentiment']['magnitude'] = \
-                        user_stack_sentiment_item.get(dep['name'], {}).get('magnitude', 0)
-                else:
-                    dep['sentiment'] = {
-                        "latest_comment": "",
-                        "overall_score": 0,
-                        "magnitude": 0
-                    }
-
-                # Adding topics from the recommendations
-                stack_recommendation = get_item_from_list_by_key_value(recommendations,
-                                                                       "manifest_file_path",
-                                                                       stack.get(
-                                                                           "manifest_file_path"))
-                if stack_recommendation is not None:
-                    dep["topic_list"] = stack_recommendation.get("input_stack_topics",
-                                                                 {}).get(dep.get('name'), [])
-                else:
-                    dep["topic_list"] = []
```

Comment:
Do we really want to remove also this populating of `topic_list` part ?

If not, then I'd suggest to move the `stack_recommendation = get_item_from_list_by_key_value(...)` outside the `for dep in user_stack_deps:` loop as it's invariant to it.

Reply:
we need ```dep[topic_list]``` so keeping this. Now, I also tried to update the suggested line and removed from the above-mentioned for loop. One more thing, i saw repetition of three lines of codes with  ```stack_recommendation = get_item_from_list_by_key_value(...)```, removed the repeated lines and appended one additional line with the updated codes. hopefully, it will have the same effect. Thanks

Reply:
I think we shouldn't simply remove the setting of `stack["recommendation"]` (as you do at the end of the `get()` method).

So what do you think about replacing  every occurrence of `stack_recommendation` with `stack["recommendation"]` ?

Reply:
I did the earlier change for ```stack_recommendation``` and ```stack["recommendation"]``` which is wrong. Both are different entities, and we should not touch them. So reverting these changes.